shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -375,6 +375,9 @@ public void onMessage(Object message, BasicMessageChannel.Reply<Object> reply) {
                 }
                 sendAccessibilityEvent(event);
                 break;
+            case "announce":
+                mOwner.announceForAccessibility((String) data.get("message"));
```

Comment:
Does/Should the message string have a text direction for RTL/LTR?

Reply:
announceForAccessibility doesn't take any parameter other than the text.
As far as I can tell the underlying AccessibilityEvent can't take direction parameter as well (https://developer.android.com/reference/android/view/accessibility/AccessibilityEvent.html#TYPE_ANNOUNCEMENT)